src/style/internal/shared_input-select-picker.scss

```diff
@@ -192,19 +190,18 @@ $height-of-helper-text-pseudo-before: 0.75rem; // There is strange a pseudo befo
         }
     }
 
-    .mdc-text-field--outlined {
-        .mdc-text-field__input {
-            padding-bottom: functions.pxToRem(12);
-        }
-    }
-
     .mdc-text-field.mdc-text-field--focused.mdc-text-field--outlined {
         .mdc-notched-outline--notched {
             .mdc-notched-outline__notch {
                 padding-top: 0;
             }
         }
     }
+
+    .mdc-text-field__icon--leading,
+    .mdc-text-field__icon--trailing {
```

Comment:
Not a big deal, but we could write this for `mdc-text-field__icon`, since both leading and trailing icons share this same class, right?

Reply:
Yeah, that makes sense 👍

Reply:
me do a fixup separately for this, so in case you have a reason for specifying these, we can drop that fixup